core/src/main/java/com/datastax/oss/driver/api/core/session/SessionBuilder.java

```diff
@@ -318,6 +319,19 @@ public SelfT addRequestTracker(@NonNull RequestTracker requestTracker) {
     return self;
   }
 
+  /**
+   * Registers a distributed trace ID generator The driver will use the distributed trace ID in the
+   * logs So that users can correlate logs about the same request from different loggers.
```

Comment:
nitpick: it looks like a period is missing here.
```suggestion
   * Registers a distributed trace ID generator. The driver will use the distributed trace ID in the
   * logs So that users can correlate logs about the same request from different loggers.
```
Also if there's nothing useful to say in `@param` and `@return` (and I would agree it is the case here), it's fine to omit them altogether.